gCoffee/app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +33,9 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
 
+            </intent-filter>
```

Comment:
If I read the pull request correct, this commit just fixes stuff from other commits of the same pull request. Is it possible to avoid such commits, e.g., is there tool support for this?

Reply:
I am planning on researching this + some potential tool for linting tomorrow